HHS/OPRE-OPS
# feat: ✨ adds active period filter to CANs list

## What changed

- adds active period filter to CANs list
- does not include tags

## Issue

#347 #2888

## How to test

1. goto CANs list
2. click on filter button
3. select an active period
4. see table update with filter
5. 💰 profit

## Screenshots

https://github.com/user-attachments/assets/ec990df8-6101-4124-84ef-03d98d05d7b1

## Definition of Done Checklist
